recipes/tipp/meta.yaml

```
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# Exit on any error
+set -ex
+
+# Define variables for directories
+# define kmc directory  
```

yamllint

[error] trailing spaces

(trailing-spaces)